communications.md

```diff
@@ -2,95 +2,62 @@
 
 ## General guidelines
 
-Communication is critical in any organization but in doing so, we want to ensure that we are considerate of other people's production processes and time. Therefore, the focus should be on real problem solving and creative endeavors instead of drowning oneself in processes and tools when communicating with one another.
+Communication is critical to any organization and in doing so, we should be considerate of other people's production processes and time.
+We use a few tools to aid customer and team (inter and intra) communication: Slack, G Suite, Asana and Emails.
 
-## First line of Communication: Texting
+Our communication tools encompasses both formal and informal communications as such we may want to understand:
 
-We use a few tools to help team members communicate with one another: Slack, G Suite and Asana. Before you decide to have a face-to-face or voice/video conference, take some time to organize your thoughts, and write them down. Revise it to be as concise as possible. If it's not urgently blocking all of your next tasks, use Slack, Gmail or Asana to contact another team member asynchronously (in an intermittent manner).
+- your audience 
+- the size of the audience 
+- content of message and reusability (if the content of the message will come in handy in the future) 
 
-Budget communication time. Try to allocate yourself a relatively fixed schedule to initiate, check and respond to messages. To a certain extent, this means try to optimize for the whole block of contiguous time that suits better for your job (knowledge-based, creative endeavors).
```

Comment:
This again is about preferring async over sync. It could be rephrased, or removed. But suggest otherwise defeats the original intent.

Reply:
Preference can be opinionated though. We can summarize this statement to: 

- optimize your time using the appropriate optimal means that works best for you.

This will create room for them to use a productive pattern that works for them. We should avoid the term **default** as well.

Reply:
This "opinion" is the bottom-line.

Reply:
Sounds like what we're trying to communicate here is:
- IF you prefer to work on contiguous blocks uninterrupted, then allocate for yourself a daily cadence (we can suggest an appropriate interval) to check and respond to messages you've received.
- IF you prefer to multi-task, then you're all good to keep comm. channels open throughout the day without sacrificing your productivity.

again, choice.